### DISCUSSION TITLE
github sign-in is blocked from Visual Studio 2022

### DISCUSSION THREAD

**Discussion by sanikolov** [community/community#121386] [Copilot Conversations]
### Select Topic Area

Bug

### Body

My github account used to be associated with a single email address.
The domain of that email address was hosted on microsoft infra until I deleted the domain.
As of today all vestiges of my domain are gone from microsoft servers.
Upon logging into github with my github credentials from Visual studio 2022 I am getting the error shown in the attached screenshot.
I speculate that for some reason github seems to think that my account belongs to an organization.
As I mentioned this is no longer true.
The upshot of the failure to log in from VS2022 is that I am unable to use copilot for which I bought a subscription.
![image](https://github.com/community/community/assets/19896827/ae6b0f8c-ebd9-4ac0-802f-f8392b8c41f7)

**Comment by Akash1134** [comment]
Hi @sanikolov ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by sanikolov** [reply]
Can somebody on the copilot side refund my copilot subscription since I am unable to make use of it.
I opened support ticket https://support.github.com/ticket/personal/0/2754293 regarding blocked access however there is no reply and based on additional searches the chances of getting a reply in the next 6 weeks are slim.
Please cancel my copilot subscription to avoid having to dispute the transaction with my CC issuer.

**Reply by sanikolov** [reply]
I am going to open a new ticket because this one is not monitored anymore.

**Comment by iTechTop** [comment]
Plz check Account Settings.
While waiting for a response from GitHub Support, you can also check your account settings on the GitHub website. Make sure that there are no unexpected settings or associations with organizations that you're unaware of.

**Comment by sanikolov** [comment]
There is no answer.
Activation of copilot license is dependent on being able to log into github from Visual Studio 2022 which doesn't work for me. Github support are not able to help me.
